app/controllers/api/internal/v1/defendants_controller.rb

```diff
@@ -36,6 +36,14 @@ def enqueue_unlink
           )
         end
 
+        def check_defendant_presence!
```

Comment:
Call `check_defendant_presence!` before `UnlinkLaaReferenceWorker.perform_async`
This is to avoid having the error inside `UnlinkLaaReferenceWorker`